root/treesitter.py

```diff
@@ -0,0 +1,48 @@
+from collections import Counter
+
+from tree_sitter import Language, Parser
+
+languages = {"python": Language("../build/my-languages.so", "python"),
```

Comment:
you need to make function that
* checks if parsers are ready
* if not - build parsers
this function should be called in the end of file - so any time you import from this file - it will be checked that parsers are ready